gc/scheduler/scheduler.go

```diff
@@ -197,6 +202,8 @@ func newScheduler(c collector, cfg *config) *gcScheduler {
 }
 
 func (s *gcScheduler) ScheduleAndWait(ctx context.Context) (gc.Stats, error) {
+	ctx, span := tracing.StartSpan(ctx, makeSpanName("ScheduleAndWait"))
```

Comment:
Do we need to create in a new span here? one of the discussion points in the image service tracing PR(7453) was that we perhaps do not need to create a new span for every method, instead we can just add additional attributes, events on the current span. Sub-spans are created mainly when we cross a component boundary.

Check @jterry75 comment on 

> As for your other question about child spans. My general philosophy (just my opinion) is that all Entry and Exit's should be covered by spans and they should flow context, at a minimum this is the span/trace id's. So what this means is that when PullImage is called we start a span. When PullImage calls into the containerd core runtime for the content store, the resolver, the layer store, the snapshotter, etc these should ALL have spans! This is the granular level that you want. Now should each of those components have spans on each private function? Likely not, but eventually we call into the shims via ttrpc and yet another span, which eventually makes some syscalls. These syscalls IMO should be surrounded by a span, so that we can measure the effect of the OS in the entire call chain. Etc. So yes, Span On Entry, Span On Exit, maybe, when needed, span in the middle.